src/ethereum/arrow_glacier/fork.py

```diff
@@ -675,6 +678,7 @@ def process_transaction(
     block_output: vm.BlockOutput,
     tx: Transaction,
     index: Uint,
+    tracer: BaseEvmTracer,
```

Comment:
We should tend towards minimizing the impact of tracing on the spec as much as possible, and I think this is going in the wrong direction. Someone calling `process_transaction` directly now has to consider tracing, where they didn't before.